src/sagas/RootSaga.js

```diff
@@ -127,8 +127,8 @@ export function* handleSaveProfile(action) {
     yield delay(300);
     yield put(saveProfileReset());
     yield put(resetDrafts());
-  } catch (e) {
-    yield put(saveProfileFailure(e.message));
+  } catch ({ fieldErrors }) {
```

Comment:
Should we just pass the whole error object along here?  Or include the non-field errors as well?  (I'm assuming that we might receive some non-field-specific errors too, sometimes...)

Reply:
If we add non-field error stuff I think we should add it to the object deconstruction. I wanted to be explicit about what we're working with. In the end, either way works.